pkg/tunnel/tunnel.go

```diff
@@ -71,7 +71,6 @@ func Client(scc node.ControllerClient, ln net.Listener, token,
 
 func connect(scc node.ControllerClient, stream net.Conn,
 	token, name string, port uint32) {
-	defer stream.Close()
```

Comment:
Why remove this?

Reply:
It's handled in `streamBidirectional`, but I realize now there are error cases that won't call that function, so we should probably leave it.